bin/lib/zopen-build

```diff
@@ -343,7 +359,7 @@ setDepsEnv()
   fi
 
   orig="${PWD}"
-  for dep in $deps; do
+  echo "$deps" | xargs | tr ' ' '\n' | sort -u | while read dep; do
```

Comment:
I believe this isn't portable.  The command on the right of the pipe may be run in a child process.  As a result all assignments in the loop will be lost after the loop.  Keep this as a for loop.